guides/source/routing.md

```diff
@@ -1108,24 +1175,19 @@ will recognize incoming paths beginning with `/photos` and route the requests to
 | PATCH/PUT | /photos/:id      | photos#update     | image_path(:id)      |
 | DELETE    | /photos/:id      | photos#destroy    | image_path(:id)      |
 
-### Overriding the `new` and `edit` Segments
+### Renaming the `new` and `edit` Helper Prefix
```

Comment:
I think this section might be wrong. As far as I could test, this _doesn't_ change the helper prefix, only the path segments. Can you double-check?

Reply:
You're right, good catch! That's too bad as the other way would be more consistent.